manifest.yml

```diff
@@ -2,6 +2,9 @@
 applications:
 - name: tock
   instances: 4
+  routes:
+  - route: tock.18f.gov
+  - route: tock.18f.gov/api
```

Comment:
Should we have these defined in here, or should they be managed elsewhere?  I ask because I've not seen the routes defined for FEC in production, I believe we use the CDN service.

Reply:
These are defined here because of how the org / space was setup in the first place for cloud.gov. The `gsa-18f-tock` org is the owner of the `18f.gov` domain and needs to have these explicitly defined in the manifest. Otherwise, CF on push attempts to assign it the default `app.cloud.gov` domain which we don't want as we use this for the staging URL right now.

Tock doesn't use the CDN service because the domain it uses is owned by the org it's being deployed in.

Additionally, the reason we're setting the `routes:` list here is because all other ways to define `host:`, `path:`, & `domain:` [have been deprecated](https://docs.cloudfoundry.org/devguide/deploy-apps/manifest.html#-attribute-routes-replaces-domain,-domains,-host,-hosts,-and-no-hostname)

Reply:
Gotcha, thank you for explaining!